packages/neon_framework/test/request_manager_test.dart

```diff
@@ -575,7 +576,10 @@ void main() {
 
       test('cached ETag', () async {
         final callback = MockCallbackFunction<CacheParameters>();
-        final newExpires = DateTime.timestamp().copyWith(microsecond: 0, millisecond: 0).add(const Duration(hours: 1));
+        final newExpires = tz.TZDateTime.from(
+          DateTime.timestamp().copyWith(millisecond: 0, microsecond: 0).add(const Duration(hours: 1)),
+          tz.UTC,
+        );
```

Comment:
Use TZDateTime and do the copyWith+add on that instead

Reply:
apparently there is no copyWith extension for TZDateTime so calling copy with on it always returns a normal `DateTime` object.
We can't add such a method to our own utils as it would conflict the default one (unless we shadow it on dart:core everywhere we use it).

As a next step (after fixing the http date parsing) I want to add some sort of normalization to our date time extensions where a `TZDateTime` object is returned